File: Accounting/Classes/Transaction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Accounting.Interfaces;

using Accounting.Models;

namespace Accounting.Classes
{
    public class Transaction
    {
        public static Models.transaction createNew(int entityID,int catTypeID,decimal amount)
        {
            using (var ctx = new AccContext())
            {
                var newTrans = new Models.transaction()
                {
                    accountID = Account.getAccount(entityID,catTypeID).ID,
                    amount = amount
                };
                ctx.transaction.AddObject(newTrans);
                ctx.SaveChanges();
                return newTrans;
            }
        }    
    }
    
}

File: projects/phpSample/Transaction.php
<?php     

namespace Accounting;

class Transaction
{
        public static function createNew($entityID, $catTypeID, $amount, $currencyID)
        {
            /*
                var newTrans = new AccountingLib.Models.transaction()
                {
                    accountID = Account.getAccount(entityID, catTypeID, currencyID).ID,
                    amount = amount
                };
                ctx.transaction.AddObject(newTrans);
                ctx.SaveChanges();
                return newTrans;                   
                */
        }    
}
